python/lsst/ts/standardscripts/auxtel/daytime_checkout/telescope_and_dome_checkout.py

```diff
@@ -0,0 +1,176 @@
+    Dome are returned to a park position and left enabled.
+    """
+
+    def __init__(self, index=1):
+
+        super().__init__(
+            index=index,
+            descr="Execute daytime checkout of AT Telescope and Dome.",
+        )
+
+        # Instantiate atcs and latiss. We need to do this after the call to
+        # super().__init__() above. We can also pass in the script domain and
+        # logger to both classes so log messages generated internally are
+        # published to the efd.
+        self.atcs = ATCS(domain=self.domain, log=self.log)
+
+    @classmethod
+    def get_schema(cls):
+        return None
+
+    async def configure(self, config):
+        # This script does not require any configuration
+        pass
+
+    def set_metadata(self, metadata):
+        """Set estimated duration of the script."""
+
+        metadata.duration = (
+            10.0 * 60
+        )  # Approximate 10min to completion, should depend on what used
+
+    async def run(self):
+
+        await self.assert_feasibility()
+
+        # Disable dome following for initial test
+        await self.atcs.disable_dome_following()
+
+        await self.checkpoint("Slewing Telecope without tracking...")
+
+        # Test point az-el
+        start_az = 0.0
+        start_el = 80.0
+        start_rot = 0
+        await self.atcs.point_azel(az=start_az, el=start_el, rot_tel=start_rot)
+
+        # Stop tracking
+        await self.atcs.stop_tracking()
+
+        await self.checkpoint("Starting tracking test...")
+
+        # Test sidereal-tracking
+        coord = self.atcs.radec_from_azel(az=start_az + 5, el=start_el - 5)
+        await self.atcs.slew_icrs(
+            coord.ra,
+            coord.dec,
+            rot=start_rot,
+            stop_before_slew=False,
+            rot_type=RotType.PhysicalSky,
+        )
+
+        # Sleep for 2 minutes to monitor sidereal tracking
+        await asyncio.sleep(2.0 * 60.0)
+
+        # Stop tracking
+        await self.atcs.stop_tracking()
+
+        await self.checkpoint("Slewing Dome...")
+
+        # Home the dome
+        self.atcs.home_dome()
+
+        # Check that Dome Moves
+        dome_az = await self.atcs.rem.atdome.tel_position.next(
+            flush=True, timeout=STD_TIMEOUT
+        )
+        self.log.info(
+            f"Dome currently thinks it is at an azimuth position of {dome_az.azimuthPosition} degrees."
+        )
+
+        d_az = 15
+        await self.atcs.rem.atdome.cmd_moveAzimuth.set_start(
+            azimuth=dome_az.azimuthPosition + d_az
+        )
+        dome_az = await self.atcs.rem.atdome.tel_position.next(
+            flush=True, timeout=STD_TIMEOUT
+        )
+        self.log.info(
+            f"After the commanded {d_az} degrees motion, the dome is at an"
+            f"azimuth position of {dome_az.azimuthPosition} degrees"
+        )
+
+        # Open and Close the Dome Shutter
+        await self.atcs.open_dome_shutter()
```

Comment:
wow! for real? This test includes opening and closing the dome shutter? during the day?

Reply:
This was not in the daytime checkout notebook, I added it after I found the function in atcs to checkout the dome shutter. I figured any moving parts that could be checkout out should be. I don't think it is unsafe during the day, the mirror is closed at this point, but I can remove it if you think it is not a good idea.

Reply:
yeah... probably remove it